recipe/meta.yaml

```diff
@@ -1,58 +1,74 @@
 {% set name = "datasets" %}
-{% set version = "1.12.1" %}
+{% set version = "2.6.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 2f3c15323dd5358184cc73e5b11e90e1e67f30c680dfc685d04903297cc09432
+  sha256: 05d27d5966f35c2d51f5dbe07b2156cc30adc1e5c744f89947d18bb8244388ea
+  patches:
+    - dill_0.3.6.patch # dill was temporarily pinned upstream to <0.3.6 due to delay in it's release. Patch to allow 0.3.6.
 
 build:
-  noarch: python
+  # s390x is missing pyarrow atm 
+  skip: true  # [py<38 or s390x] 
```

Comment:
We should skip `py<37`, see https://github.com/huggingface/datasets/blob/1742cf16125d17a2420819fd599278484c915364/setup.py#L210.

Reply:
Thanks! I've updated that.